.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,20 @@
+name: Pull request check
+
+on:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.10.0
+
+      - name: Build
```

Comment:
```suggestion
      - name: CI
```

Reply:
This step actually runs only `npm run build`, so it's best to leave it as is.